py-polars/polars/internals/dataframe/frame.py

```diff
@@ -3765,6 +3765,7 @@ def groupby_dynamic(
         - 1d    (1 day)
         - 1w    (1 week)
         - 1mo   (1 calendar month)
+        - 1q    (1 quarter)
```

Comment:
can you put this in all the relevant docs where duration strings are listed?

Reply:
done.